partner_communication_switzerland/data/sponsorship_communications_cron.xml

```diff
@@ -40,5 +31,88 @@
+            <field name="trigger">on_time</field>
+            <field name="trg_date_id">start_date</field>
+            <field name="trg_date_range">36</field>
+            <field name="trg_date_range_type">month</field>
+            <field name="active" eval="False"/>
+        </record>
+        <record id="sponsorship_anniversary_5_communication" model="base.automation">
+            <field name="name">Send sponsorship anniversary 5 years communication</field>
+            <field name="model_id" ref="model_recurring_contract"/>
+            <field name="config_id" ref="planned_anniversary_5"/>
+            <field name="state">communication</field>
+            <field name="partner_field">correspondent_id</field>
+            <field name="filter_domain">[("start_date", ">=", Datetime.now() - relativedelta(years=5))]</field>
+            <field name="trigger">on_time</field>
+            <field name="trg_date_id">start_date</field>
+            <field name="trg_date_range">60</field>
+            <field name="trg_date_range_type">month</field>
+            <field name="active" eval="False"/>
+        </record>
+        <record id="sponsorship_anniversary_10_communication" model="base.automation">
+            <field name="name">Send sponsorship anniversary 10 years communication</field>
+            <field name="model_id" ref="model_recurring_contract"/>
+            <field name="config_id" ref="planned_anniversary_10"/>
+            <field name="state">communication</field>
+            <field name="partner_field">correspondent_id</field>
+            <field name="filter_domain">[("start_date", ">=", Datetime.now() - relativedelta(years=10))]</field>
+            <field name="trigger">on_time</field>
+            <field name="trg_date_id">start_date</field>
+            <field name="trg_date_range">120</field>
+            <field name="trg_date_range_type">month</field>
+            <field name="active" eval="False"/>
+        </record>
+        <record id="sponsorship_anniversary_15_communication" model="base.automation">
+            <field name="name">Send sponsorship anniversary 15 years communication</field>
+            <field name="model_id" ref="model_recurring_contract"/>
+            <field name="config_id" ref="planned_anniversary_15"/>
+            <field name="state">communication</field>
+            <field name="partner_field">correspondent_id</field>
+            <field name="filter_domain">[("start_date", ">=", Datetime.now() - relativedelta(years=15))]</field>
+            <field name="trigger">on_time</field>
+            <field name="trg_date_id">start_date</field>
+            <field name="trg_date_range">180</field>
+            <field name="trg_date_range_type">month</field>
+            <field name="active" eval="False"/>
+        </record>
+
+        <!-- Write and pray reminders -->
+        <t t-set="min_date" t-value="Datetime.now() - relativedelta(months=3)"/>
```

Comment:
This could be done directly in the `filter_domain` field like you did for other rules